blocks/search/search.css

```diff
@@ -1,106 +1,113 @@
 .search__box {
```

Comment:
this file has the most significant refactor; let me know if you'd like to talk through it